web/api/netdata-swagger.json

```diff
@@ -1220,6 +1220,11 @@
             "description": "Container technology detection method.",
             "example": "dockerenv"
           },
+          "stream_compression": {
+            "type": "boolean",
+            "description": "Stream transmission compression method.",
+            "example": "true"
```

Comment:
_for description -> Doesn't it mean compression is enabled?_
It means if the compression is enabled or not for the sender thread (as you stated) of host. Obviously it can be both `disabled `(`false`) or `enabled `(`true`). I can write more detail in the description section. 

Before the PR `null` had only one mean: agent built without stream compression support. In the case of building with compression support it should be `enabled` or `disabled` by the user. But there is a third case which agent is built with compression support but the `sender state` is unreachable (related pointer is `null`). Therefore we can return a different value for this state. My suggestion is below, maybe we can swap `N/A` and `NULL` if it is more meaningful. 

![image](https://user-images.githubusercontent.com/51407/160908985-848372b2-887e-4676-acf9-b1d63c3a3bc0.png)

What do you think?

Reply:
lets not overcomplicate this then, to fix the segfault quickly, we can improve in future